### PR TITLE
Hypershift density tests

### PR DESCRIPTION
**Pull request by morenod**
Add function for loading cluster with e2e-benchmarking executions

### CODE REVIEW

File: hypershift/hosted-wrapper.py [hunk 1]
```diff
@@ -567,9 +674,9 @@ def main():
         watcher_cleanup.run = False
         watcher.join()
 
-    if args.cleanup:
-        logging.info('Cleaning working directory %s' % my_path)
-        shutil.rmtree(my_path)
+    # if args.cleanup:
```
Comment (dry923) [inline hypershift/hosted-wrapper.py:677]: why is this commented out?

Comment (mukrishn) [inline hypershift/hosted-wrapper.py:677]: @morenod qq is this change intentional?

Comment (mukrishn) [reply hypershift/hosted-wrapper.py:677]: otherwise LGTM

**Review by mukrishn — APPROVED** [review]
LGTM
